### ISSUE TITLE
LibreOffice AddIn

### ISSUE THREAD

**Issue by j-dimension** [jlawyerorg/j-lawyer-org#81]

**Comment by j-dimension** [comment]
Siehe #574

**Comment by PBaumfalk** [comment]
Nur zur Info und damit es nicht verloren geht: Während meiner Recherche ist mir auf macOS aufgefallen: Die Version von LibreOffice, die zur Zeit auf Mac genutzt wird, wegen **unoconv**, die hat Probleme mit dem Import von Extensions. Vielleicht müsste man vorher das unoconv Problem auf Mac lösen und auf die neueste LibreOffice Version updaten, bevor man eine richtige Extension schreibt.

**Comment by j-dimension** [comment]
Genau, das wäre der richtige Weg!

**Comment by j-dimension** [comment]
https://github.com/unoconv/unoconv/issues/263

**Comment by j-dimension** [comment]
@PBaumfalk  was hätte eine "richtige" Extension für Vorteile gegenüber dem hier: https://github.com/jlawyerorg/j-lawyer-documentation/issues/1

Da mit der 1.12 dann unter macOS eine aktuelle LibreOffice-Version genutzt werden kann - wäre eine Extension evtl. die bessere Lösung?

**Comment by PBaumfalk** [comment]
Ja, hätte es. Muss nur herausfinden, wie man eine Extension erstellt...

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by PBaumfalk** [comment]
Und eine wesentlich schnellere Installation.

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by PBaumfalk** [comment]
https://amantke.de/wp-content/uploads/2017/07/extensionsbook2017.pdf

**Comment by j-dimension** [comment]
Siehe #751

**Comment by librelegal** [comment]
> unoconv/
[Unoconv is deprecated](https://github.com/unoconv/unoconv#unoconv-is-deprecated)
https://github.com/unoconv/unoserver/
